stem/water_boundaries.py

```diff
@@ -0,0 +1,105 @@
+from typing import List, Union
+from dataclasses import dataclass, field
+from abc import ABC
+
+
+
+@dataclass
+class WaterBoundaryParameters(ABC):
+    """
+    Abstract base class for load water boundary parameters
+
+    Attributes:
+        - surfaces_assigment (List[str]): List of surfaces to which the water boundary is assigned.
+        - is_fixed (bool): True if the water boundary is fixed, False otherwise.
+        - gravity_direction (int): Direction of the gravity vector.
+        - out_of_plane_direction (int): Direction of the out of plane vector.
+
+
+    """
+    surfaces_assigment: List[str] = field(default_factory=lambda: [""])
```

Comment:
i think the surface assignment will be managed in the modelparts. In this file I propose to only put the parameters as present in the projectparameters.json

Reply:
in the future, we should add some helper functions in model.py, which should handle this. But I would say thats a problem for later